### PR TITLE
Opinions

### PR DESCRIPTION
**Pull request by cghost420**
What about this?

### CODE REVIEW

**Comment by TallTed** [comment]
> What about this?

Are you now a member of the W3C Verifiable Credentials Working Group, and therefore able to make substantive contributions (which this may or may not be; I have not evaluated it)?

What issue are you trying to address with this pull request?

**Comment by cghost420** [comment]
I give up idk how to do this.

File: .github/workflows/greetings.yml [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+jobs:
+  greeting:
+    runs-on: ubuntu-latest
+    permissions:
+      issues: write
+      pull-requests: write
+    steps:
+    - uses: actions/first-interaction@v1
+      with:
+        repo-token: ${{ secrets.GITHUB_TOKEN }}
```
Comment (OR13) [inline .github/workflows/greetings.yml:14]: This doesn't actually look that bad, on second view: https://github.com/actions/first-interaction

**Comment by OR13** [comment]
I reviewed these changes,  they don't seem malicious.

I don't think this repo needs them though.

**Comment by cghost420** [comment]
Bro I don't know how to do this. I've tried and all that happens is me
fucking shit up. Someone else can take over.

**Comment by cghost420** [comment]
I can't figure github out.

**Pull request closed without merging**